tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -43,6 +45,7 @@
 
 <dom-module id="tf-profile-dashboard">
 <template>
+<paper-toast id="toast" text="" always-on-top></paper-toast>
```

Comment:
DOM-wise, you might want at the end so it shows up on top of other sibling nodes.